### ISSUE TITLE
[Feature Request] Add ParaLLel-N64 to the RetroArch WebPlayer

### ISSUE THREAD

**Issue by hyarsan** [libretro/RetroArch#5613]
I am not sure if this should be on ParaLLel-N64's repository but...
can ParaLLel-N64 be added to the list of cores in the WebPlayer?

**Comment by hizzlekizzle** [comment]
I don't think we were able to get the webplayer to cooperate with GL-based cores.

**Comment by hyarsan** [comment]
But why does Mupen64Plus exist on the WebPlayer then? Or Craft...

**Comment by twinaphex** [comment]
It worked before. The single thread code paths has probably long since ceased to work. 

Problem is the web player is in its infancy, the menu toggle button does not even work properly. I pointed this out many times to my team members, nobody ever did anything about it.

**Comment by hyarsan** [comment]
Yeah the menu toggle does not work anymoreq. I thought that it was just me!

**Comment by bparker06** [comment]
@twinaphex The menu toggle *was* technically fixed, the currently problem seems like a browser issue with handling the F1 key. If you set the menu toggle to something else (and the button on the webpage), it works fine.

**Comment by hyarsan** [comment]
What about angrylion? I thought it worked for all the video drivers? It would still be pretty slow though even if it's multi-threaded....

**Comment by twinaphex** [comment]
Cached interpreter with angrylion could work, HOWEVER, we need a non-libco codepath, libco can't be made to work with Emscripten.
